source/include/core_http_client.h

```diff
@@ -531,8 +556,12 @@ HTTPStatus_t HTTPClient_InitializeRequestHeaders( HTTPRequestHeaders_t * pReques
  * bytes written.
  *
  * Headers are written in the following format:
- *     <pName>: <pField>\r\n\r\n
- * The trailing \r\n that denotes the end of the header lines is overwritten,
+ *
+ * @code
+ *     <field>: <value>\r\n\r\n
+ * @endcode
+ *
+ * The trailing \\r\\n that denotes the end of the header lines is overwritten,
```

Comment:
Should these be wrapped in <code></code> for clarity?

Reply:
sure

Reply:
Actually, `\r\n` might work too and would look better